denseclus/DenseClus.py

```diff
@@ -254,6 +296,32 @@ def fit(self, df: pd.DataFrame) -> None:
         logger.info("Fitting HDBSCAN...")
         self._fit_hdbscan()
 
+    @staticmethod
```

Comment:
@bharven Can we have it so that it uses `self` and is a class method? Static makes it so that it might as well be a function.